packages/lib-runtime/src/store/PluginStoreContext.tsx

```diff
@@ -14,19 +16,38 @@ export const PluginStoreProvider: React.FC<PluginStoreProviderProps> = ({ store,
   return <PluginStoreContext.Provider value={store}>{children}</PluginStoreContext.Provider>;
 };
 
-type PluginStoreProviderProps = React.PropsWithChildren<{
+export type PluginStoreProviderProps = React.PropsWithChildren<{
   store: PluginStore;
 }>;
 
 /**
- * React hook for consuming the `PluginStore` via its client interface.
+ * React hook for exposing the {@link PluginConsumer} interface.
  */
-export const usePluginStore = (): PluginStoreClient => {
+export const usePluginConsumer = (): PluginConsumer => {
   const store = React.useContext(PluginStoreContext);
 
   if (store === undefined) {
-    throw new Error('usePluginStore hook called outside a PluginStoreProvider');
+    throw new Error('usePluginConsumer hook called outside a PluginStoreProvider');
   }
 
-  return store;
+  return React.useMemo(
+    () => createMethodDelegate(store, ['subscribe', 'getExtensions', 'getPluginInfo']),
+    [store],
+  );
+};
+
+/**
+ * React hook for exposing the {@link PluginManager} interface.
+ */
+export const usePluginManager = (): PluginManager => {
+  const store = React.useContext(PluginStoreContext);
+
+  if (store === undefined) {
```

Comment:
```suggestion
  if (store == null) {
```
or
```suggestion
  if (!store) {
```

Reply:
Also missed here that context has type `| undefined`.

Reply:
Similar to above, I'll update the check to be an `instanceof` check (which also returns `false` on `null` or `undefined` values).